### DISCUSSION TITLE
Welcome to silero-models Discussions!

### DISCUSSION THREAD

**Discussion by snakers4** [snakers4/silero-models#34] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Abhi011999** [comment]
Hi @snakers4 !

Just wanted to ask, when you will make the model code public (if you are ever going to) ? It will surely help a lot.

**Comment by snakers4** [comment]
Hi @Abhi011999!

Model code is already public

**Reply by Abhi011999** [reply]
Oh, I thought you kept it private.
Can I get a link or reference of some sort for the silero model structure's code?

I want to do some research on it and see how it actually compares with modified DeepSpeech2.

**Comment by snakers4** [comment]
> Can I get a link or reference of some sort for the silero model structure's code?

take any jit container https://github.com/snakers4/silero-models/blob/master/models.yml#L55
